health/probe/plugin_impl_probe.go

```diff
@@ -101,8 +113,8 @@ func (p *Plugin) readinessProbeHandler(formatter *render.Render) http.HandlerFun
 // livenessProbeHandler handles k8s liveness probe.
 func (p *Plugin) livenessProbeHandler(formatter *render.Render) http.HandlerFunc {
 	return func(w http.ResponseWriter, req *http.Request) {
-		stat := p.StatusCheck.GetAgentStatus()
-		statJSON, _ := json.Marshal(p.StatusCheck.GetAgentStatus())
+		stat := p.getAgentStatus()
+		statJSON, _ := json.Marshal(stat)
```

Comment:
any reason to ignore error here?

Reply:
It was ignored before and I'm not sure whether it's worth to log it or do something about it. Any suggestion?